### ISSUE TITLE
Ability to change Mouse Tooltips to the right of your cursor?

### ISSUE THREAD

**Issue by CaraPis** [runelite/runelite#1056]
Hello, 

Right now the tooltip comes up at the top left of your cursor: https://i.imgur.com/VQqKCbS.png
I found this pretty annoying to be honest since it often blocks me from seeing what's behind my cursor.
Could we have an option to move the tooltip to the middle right/right bottom side of the mouse (like on OSBuddy/Runeloader)? I believe this would solve a lot of the "blocking". Also, I found the tooltips pretty much only useful for in the "map area" as i would call it (https://i.imgur.com/7eBlLOq.png). I think it gets a little cluttered otherwise.

Regards, Cara.

**Comment by sethtroll** [comment]
#959 does what I think you want.

**Comment by SomeoneWithAnInternetConnection** [comment]
I don't think it covers the bundled request to only show tooltips on things in the viewport though. I'm not entirely sure I'd be keen on a change like that though; clipping all the tooltips to that region would basically remove item stats tooltips entirely, and in general the mouse highlight tooltips make it obvious when things are clickable (similar to the hand cursor in the rest of computer world), which is kinda nice

**Comment by deathbeam** [comment]
#959 solves the first part of the issue. For second part, we can add toggle to not show the "widget" tooltips.

**Comment by CaraPis** [comment]
Ok, should i remove this request then and make a new one with only the second part of the request in it?

**Comment by deathbeam** [comment]
No, the PR is still open. But this should definitely be split into 2 separate issues.
